src/features/dashboard/dashboard.tsx

```diff
@@ -16,7 +17,7 @@ const Dashboard = () => {
   }, [setIsDashboard]);
 
   if (is_logged_in) return <ManageDashboard />;
-  return <Login />;
+  return <BrowserOnly>{() => <Login />}</BrowserOnly>;
```

Comment:
[Q] is this flow update expected?

Reply:
Yeap cause we are using the useHandleLogin hook inside the Login component